### ISSUE TITLE
How to render multiple objects in one image?

### ISSUE THREAD

**Issue by liyi14** [glumpy/glumpy#126]
Hello.
I have several models and want to render them in one image.
But I haven't seen any demo dealing with multiple objects.
Is it possible to use glumpy to do so?
Thanks.

**Comment by rougier** [comment]
Yes, you can. If your objects are simple enoug you can use collections, else you have to use several programs.

**Comment by liyi14** [comment]
Thanks for your reply.
I am not sure whether my case is simple or complex, they are traditional object models with .obj format.
Will it be possible to use collections? Or will it be possible to have multiple m_model, m_view, v_texcoord or things like that in the vertex code?
If not, how to deal with occlusions and interactions if use several programs?

**Comment by rougier** [comment]
Yes and no. If they're not textured, it might be possible but this may require some work.
For occlusions, it should just work (OpenGL takes care of that) but for interaction, it depends what you mean by that.

Do you have an example somewhere ?
